### DISCUSSION TITLE
How to find remaining page space using mpdf

### DISCUSSION THREAD

**Discussion by sajidanwar2020** [mpdf/mpdf#1443] [Q&A]
I want to find remaining page space for my content. If content cannot comes fit in the page then I want to move the content next page.
Actually i am showing wordpress posts.  In some cases my some content comes in upper page and some goes to next page.  i want to show my posts  in one page not to separate pages

[Stackoverflow link](http://stackoverflow.com/questions/43229693/how-to-find-remaining-page-space-using-mpdf)

**Comment by yasirdocs** [comment]
you can use, page breaks

page-break-before: always|right|left

or 

you can read this :
https://mpdf.github.io/paging/using-page.html

**Comment by nsmith-sas** [comment]
You can use this function $mpdf->_getHtmlHeight($html) is included in 6.1 and higher

**Comment by marclaporte** [comment]
Related:
https://github.com/mpdf/mpdf/issues/48
https://dev.tiki.org/Widows-and-Orphans-in-mPDF

This is very important for us:
http://wikisuite.org/blogpost5-Tiki17-makes-collaborative-digital-publishing-a-breeze
